src/test/java/pl/touk/sputnik/connector/http/SslVerificationTest.java

```diff
@@ -88,49 +96,49 @@ public void verifySslTrust() throws Exception {
     }
 
     @Test
-    public void verifySslTrustThrowsSSLHandshakeException() throws Exception {
+    void verifySslTrustThrowsSSLHandshakeException() throws Exception {
         // given
-        ConnectorDetails connectorDetails = buildConnectorDetails("localhost", untrustedService.httpsPort(), "true", "true");
+        ConnectorDetails connectorDetails = buildConnectorDetails("localhost", untrustedServer.httpsPort(), "true", "true");
         HttpHelper httpHelper = new HttpHelper();
         HttpHost httpHost = httpHelper.buildHttpHost(connectorDetails);
         CloseableHttpClient closeableHttpClient = httpHelper.buildClient(httpHost, connectorDetails);
 
         // when
-        when(closeableHttpClient).execute(httpHost, new HttpGet("/hello"));
+        Throwable thrown = catchThrowable(() -> closeableHttpClient.execute(httpHost, new HttpGet("/hello")));
 
         // then
-        then(caughtException()).isInstanceOf(SSLHandshakeException.class);
+        assertThat(thrown).isInstanceOf(SSLHandshakeException.class);
     }
 
     @Test
-    public void useDefaultTrustStoreToVerifySslTrust() throws Exception {
+    void useDefaultTrustStoreToVerifySslTrust() throws Exception {
         // given
-        ConnectorDetails connectorDetails = buildConnectorDetails("localhost", untrustedService.httpsPort(), "true", "true");
+        ConnectorDetails connectorDetails = buildConnectorDetails("localhost", untrustedServer.httpsPort(), "true", "true");
         HttpHelper httpHelper = new HttpHelper();
         HttpHost httpHost = httpHelper.buildHttpHost(connectorDetails);
         CloseableHttpClient closeableHttpClient = httpHelper.buildClient(httpHost, connectorDetails);
 
         // when
-        when(closeableHttpClient).execute(httpHost, new HttpGet("/hello"));
+        Throwable thrown = catchThrowable(() -> closeableHttpClient.execute(httpHost, new HttpGet("/hello")));
 
         // then
-        then(caughtException()).isInstanceOf(SSLHandshakeException.class);
+        assertThat(thrown).isInstanceOf(SSLHandshakeException.class);
     }
 
     @Test
-    public void verifyHostnameThrowsSSLPeerUnverifiedExceptionWhenHostDoesNotMatch() throws Exception {
+    void verifyHostnameThrowsSSLPeerUnverifiedExceptionWhenHostDoesNotMatch() throws Exception {
         // given
         setSystemTrustStore();
-        ConnectorDetails connectorDetails = buildConnectorDetails("127.0.0.1", trustedService.httpsPort(), "true", "true");
+        ConnectorDetails connectorDetails = buildConnectorDetails("127.0.0.1", trustedServer.httpsPort(), "true", "true");
```

Comment:
\[PMD\] INFO: Do not hard code the IP address

Reply:
Done